src/hooks/custom-hooks/useSendbird.ts

```diff
@@ -133,7 +133,6 @@ const useSendbird = (orderId: string | undefined, isErrorOrderInfo: boolean, cha
 
             const shouldSortFromMostRecent = messages ? messages?.length > 0 : false;
             const retrievedMessages = await channel.getMessagesByTimestamp(timestamp, {
-                customTypesFilter: [''],
```

Comment:
We technically dont need any filters here, we should just allow all messages to come in. Because mobile team add a custom filter for order_id, i tried to check for it, but it didnt work exactly 😓 but removing this does